data/human/outfits.txt

```diff
@@ -291,7 +291,8 @@ outfit "Small Outfits Expansion"
 	"cooling ineffciency" .5
 	"heat dissipation" -.005
 	description "This expansion is a smaller version of the outfit expansion."
```

Comment:
I forgot the unplunderable.

Reply:
Looks like a typo: "cooling ineffciency" .5
Should be "inefficiency"?